packages/graphback-runtime/src/data/RESTDataProvider.ts

```diff
@@ -0,0 +1,129 @@
+import { GraphbackDataProvider } from './GraphbackDataProvider';
+import { GraphQLObjectType } from 'graphql';
+const fetch = require('node-fetch');
+
+export class RESTDataProvider<Type = any, GraphbackContext = any> implements GraphbackDataProvider<Type, GraphbackContext>{
+    // Base url of the endpoint. When creating an RESTDataProvider instance, a string is passed into the constructor.
+    protected baseUrl : string;
+    protected baseType : GraphQLObjectType;
+    protected headers : any;
+
+    //constructor
+    public constructor(baseType: GraphQLObjectType, baseUrl:string, headers:any) {
+      this.baseUrl = baseUrl;
+      this.baseType = baseType;
+      this.headers = headers;
+    }
+
+    /** 
+      @param data - JSON object which carries the data needed for the update operation
+     * @param context -passed by the resolvers
+     * 
+     * valid url => (baseUrl+path) 
+     * baseUrl is defined at instance creation via the constructor. path is created from baseType name and other passed values.
+     * 
+     */
+    async create(data: Type, context?: GraphbackContext): Promise<Type> {
+        console.log(this.baseType.name.toLocaleLowerCase())
+        const url = `${this.baseUrl}/${this.baseType.name.toLowerCase()}/`   
+        const res = await fetch(url, {
+            method: 'post',
+            body:    JSON.stringify(data),
+            headers: this.headers,
+        })
+        const json = await res.json();
+        return json;
+    }
+
+
+    /**
+     * 
+     * @param data - JSON object which carries the data needed for the update operation
+     * @param context -passed by the resolvers
+     * 
+     * valid url => (baseUrl+path) 
+     * baseUrl is defined at instance creation via the constructor. path is created from baseType name and other passed values.
+     * 
+     */
+    async update(data: Type, context?: GraphbackContext): Promise<Type> {
+        const url = this.baseUrl+`/${this.baseType.name.toLocaleLowerCase}`+`/${data['id']}`
```

Comment:
Also it looks like we forgot about calling method, which makes me think about unit tests handling this case :D

```suggestion
        const url = this.baseUrl+`/${this.baseType.name.toLocaleLowerCase()}`+`/${data['id']}`
```